frankenphp_test.go

```diff
@@ -176,6 +176,53 @@ func testServerVariable(t *testing.T, opts *testOptions) {
 	}, opts)
 }
 
+func TestUnusualServerVariable_module(t *testing.T) {
```

Comment:
I wonder if the result of this test would be the same if it was done with caddy_test